app/routes/interestgroups/components/InterestGroupMemberList.css

```diff
@@ -32,3 +40,31 @@
 .row i {
   margin-right: 5px;
 }
+
+.leader {
+  font-weight: 900;
+  text-shadow: 0.0625em 0.125em 0.875em var(--goldblur);
+  color: var(--leadergold);
+}
+
+.coleader {
+  font-weight: 750;
+  text-shadow: 0.0625em 0.125em 0.875em var(--silverblur);
+  color: var(--coleadersilver);
```

Comment:
Please add some hyphens on these variables - coleadersilver was kinda difficult to read 😅